N/Notcurses/build_tarballs.jl

```diff
@@ -1,64 +1,88 @@
 
 if [[ $target == *mingw* ]]; then
     export CFLAGS="${CFLAGS} -D_WIN32_WINNT=0x0600"
-    cp ${WORKSPACE}/srcdir/headers/pthread_time.h "/opt/${target}/${target}/sys-root/include/pthread_time.h"
+    cp ${WORKSPACE}/srcdir/headers/pthread_time.h /opt/${target}/${target}/sys-root/include/pthread_time.h
 fi
 
-install_license COPYRIGHT
-
-mkdir build && cd build
-
-export CFLAGS="${CFLAGS} -I${includedir}"
+multimedia=ffmpeg
+if [[ ${bb_full_target} == armv6l-* ]]; then
+    # FFMpeg is not available on armv6l
+    multimedia=none
+elif [[ ${target} == *mingw* ]]; then
+    # FFMpeg is not found (why?)
+    # We patch CMakelists.txt for shared libraries in Windows, maybe this goes wrong?
+    multimedia=none
+fi
 
 FLAGS=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
        -DCMAKE_INSTALL_PREFIX=${prefix}
        -DCMAKE_BUILD_TYPE=Release
+       -DBUILD_EXECUTABLES=ON
        -DBUILD_SHARED_LIBS=ON
-       -DUSE_DOCTEST=off
-       -DUSE_PANDOC=off
-       -DUSE_STATIC=off
-       -DUSE_QRCODEGEN=off
-       -DBUILD_EXECUTABLES=off
-       -DUSE_POC=off
-       -DUSE_MULTIMEDIA=none
-       -DUSE_CXX=off
+       -DUSE_CXX=OFF
+       -DUSE_DOCTEST=OFF
+       -DUSE_MULTIMEDIA=${multimedia}
+       -DUSE_PANDOC=OFF
+       -DUSE_POC=OFF
+       -DUSE_QRCODEGEN=OFF
+       -DUSE_STATIC=OFF
        )
 
-cmake .. "${FLAGS[@]}"
-make -j${nproc}
-make install
+if [[ ${target} == aarch64-apple-* ]]; then
+    # Linking libomp requires the function `__divdc3`, which is implemented in
+    # `libclang_rt.osx.a` from LLVM compiler-rt.
+    FLAGS+=(-DCMAKE_SHARED_LINKER_FLAGS="-L${libdir}/darwin -lclang_rt.osx"
+            -DCMAKE_EXE_LINKER_FLAGS="-L${libdir}/darwin -lclang_rt.osx"
+            )
+fi
+
+cmake -B build "${FLAGS[@]}"
+cmake --build build --parallel ${nproc}
+cmake --install build
+install_license COPYRIGHT
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line.
-platforms = supported_platforms(; experimental=true)
+platforms = supported_platforms()
 
 # The products that we will ensure are always built.
 products = [
-    LibraryProduct("libnotcurses", :libnotcurses)
-    LibraryProduct("libnotcurses-core", :libnotcurses_core)
-    LibraryProduct("libnotcurses-ffi", :libnotcurses_ffi)
+    ExecutableProduct("notcurses-demo", :notcurses_demo),
+    ExecutableProduct("notcurses-info", :notcurses_info),
+    LibraryProduct("libnotcurses", :libnotcurses),
+    LibraryProduct("libnotcurses-core", :libnotcurses_core),
+    LibraryProduct("libnotcurses-ffi", :libnotcurses_ffi),
 ]
 
 # Dependencies that must be installed before this package can be built.
+llvm_version = v"13.0.1+1"
 dependencies = [
+    Dependency("FFMPEG_jll"),
     Dependency("Ncurses_jll"),
-    Dependency("libunistring_jll"),
     Dependency("libdeflate_jll"),
+    Dependency("libunistring_jll"),
+    # We need libclang_rt.osx.a for linking libomp, because this library provides the
```

Comment:
The reference to libomp here and above is misleading.

Reply:
Right, this is about `FFMPEG`.